src/admin-guide/environments/basic.yml

```
@@ -0,0 +1,134 @@
+name: cylc-8-basic
+# Cylc 8 without GUI installed from Conda Repos.
+channels:
+  - defaults
+  - conda-forge
+  - kinow
```

We only need the conda-forge channel (all recipes).

```suggestion
channels:
  - conda-forge
```